tests/unittest/_torch/speculative/hw_agnostic/test_save_state.py

```
@@ -50,24 +48,25 @@ def test_multi_save_state():
+            eagle3_layers_to_capture=layers_to_capture,
+        )
 
         llm_spec = LLM(**llm_common_config, speculative_config=spec_config)
 
         tok_ids = llm_spec.tokenizer.encode("The future of AI is")
 
         sampling_params = SamplingParams(max_tokens=32, temperature=0)
-        for output in llm_spec.generate_async(tok_ids,
-                                              sampling_params,
-                                              streaming=True):
+        for output in llm_spec.generate_async(tok_ids, sampling_params, streaming=True):
```

Ruff (0.15.12)

[warning] Loop control variable `output` not used within loop body

Rename unused `output` to `_output`

(B007)